meta-xt-rpi5-prod-devel-domu/recipes-kernel/linux/linux-generic-armv8.bbappend

```diff
@@ -0,0 +1,10 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/files/:"
+
+BRANCH = "master"
+SRCREV = "6613476e225e090cc9aad49be7fa504e290dd33d"
+LINUX_VERSION = "6.8.0-rc1"
+
+SRC_URI = " \
+    git://github.com/torvalds/linux.git;branch=${BRANCH};protocol=https \
+    file://defconfig \
+  "
```

Comment:
It could be better to move `"` to the start of the line or align it with the previous line.